tests/data_files/Makefile

```diff
@@ -0,0 +1,57 @@
+OPENSSL = openssl
+
+cli_crt_key_file_rsa = cli-rsa.key
+cli_crt_extensions_file = cli.opensslconf
+test_ca_key_file_rsa = test-ca.key
+test_ca_pwd_rsa = PolarSSLTest
+test_ca_config_file = test-ca.opensslconf
+
+default: all_final
+
+all_intermediate := # temporary files
+all_final := # files used by tests
+
+test-ca.csr: $(test_ca_key_file_rsa) $(test_ca_config_file)
+	$(OPENSSL) req -new -config $(test_ca_config_file) -key $(test_ca_key_file_rsa) -passin "pass:$(test_ca_pwd_rsa)" -subj "/C=NL/O=PolarSSL/CN=PolarSSL Test CA" -out $@
+all_intermediate += test-ca.csr
```

Comment:
From a functional point of view, there is no problem. I was just curious, I personally find it easier to read if everything is assigned at the top of the file. However, this is just a matter of coding style and there is nothing in ours that says this is not allowed.